src/openrct2/ride/Vehicle.h

```diff
@@ -150,6 +150,28 @@ enum VEHICLE_STATUS
 
 struct rct_vehicle_sound_params;
 
+enum VEHICLE_TRACK_SUBPOSITION : uint8_t
+{
+    VEHICLE_TRACK_SUBPOSITION_0,
+    VEHICLE_TRACK_SUBPOSITION_CHAIRLIFT_1,
+    VEHICLE_TRACK_SUBPOSITION_CHAIRLIFT_2,
+    VEHICLE_TRACK_SUBPOSITION_CHAIRLIFT_END_3,
+    VEHICLE_TRACK_SUBPOSITION_CHAIRLIFT_END_4,
+    VEHICLE_TRACK_SUBPOSITION_GO_KART_5,
```

Comment:
5 left track, 6 right track, 7 Move from left to right, 8 Move from right to left